src/main/resources/hudson/plugins/throttleconcurrents/ThrottleJobProperty/config.jelly

```diff
@@ -20,6 +20,18 @@
              field="maxConcurrentPerNode">
       <f:textbox />
     </f:entry>
+
+    <f:optionalBlock name="limitOneJobWithMatchingParams"
+                     title="${%Prevent multiple jobs with identical parameters from running concurrently}"
+                     inline="true"
+                     checked="${instance.limitOneJobWithMatchingParams}">
+      <f:entry title="${%List of parameters to check (comma-separated)}"
+               field="paramsToUseForLimit"
+               help="${descriptor.getHelpFile('limitOneJobWithMatchingParams')}">
+        <f:textbox />
```

Comment:
like this: 
```
<f:optionalBlock field="limitOneJobWithMatchingParams"
                     title="${%Prevent multiple jobs with identical parameters from running concurrently}"
                     inline="true">
      <f:entry title="${%List of parameters to check (comma-separated)}"
               field="paramsToUseForLimit">
        <f:textbox />
```

You might be interested in the implementation of `optionalBlock`: https://github.com/jenkinsci/jenkins/blob/jenkins-1.596.1/core/src/main/resources/lib/form/optionalBlock.jelly